src/app/pages/MostReadPage/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,1144 @@
+    />
+    <link
+      href="https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-192x192.png"
+      rel="icon"
+      sizes="192x192"
+      type="image/png"
+    />
+    <link
+      href="https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-512x512.png"
+      rel="apple-touch-startup-image"
+    />
+    <link
+      href="/favicon.ico"
+      rel="shortcut icon"
+      type="image/x-icon"
+    />
+    <meta
+      content="IE=edge"
+      http-equiv="X-UA-Compatible"
+    />
+    <meta
+      charset="utf-8"
+    />
+    <meta
+      content="noodp,noydir"
+      name="robots"
+    />
+    <meta
+      content="#B80000"
+      name="theme-color"
+    />
+    <meta
+      content="width=device-width, initial-scale=1, minimum-scale=1"
+      name="viewport"
+    />
+    <meta
+      content="BBC News Pidgin"
+      name="apple-mobile-web-app-title"
+    />
+    <meta
+      content="BBC News Pidgin"
+      name="application-name"
+    />
+    <meta
+      content="mostread"
+      name="description"
+    />
+    <meta
+      content="100004154058350"
+      property="fb:admins"
+    />
+    <meta
+      content="1609039196070050"
+      property="fb:app_id"
+    />
+    <meta
+      content="yes"
+      name="mobile-web-app-capable"
+    />
+    <meta
+      content="#B80000"
+      name="msapplication-TileColor"
+    />
+    <meta
+      content="https://news.files.bbci.co.uk/include/articles/public/pidgin/images/icons/icon-144x144.png"
+      name="msapplication-TileImage"
+    />
+    <meta
+      content="mostread"
+      property="og:description"
+    />
+    <meta
+      content="https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"
+      property="og:image"
+    />
+    <meta
+      content="BBC News Pidgin"
+      property="og:image:alt"
+    />
+    <meta
+      content="pcm"
+      property="og:locale"
+    />
+    <meta
+      content="BBC News Pidgin"
+      property="og:site_name"
+    />
+    <meta
+      content="De one we dem de read well well - BBC News Pidgin"
+      property="og:title"
+    />
+    <meta
+      content="website"
+      property="og:type"
+    />
+    <meta
+      content="https://www.test.bbc.com/pathname"
+      property="og:url"
+    />
+    <meta
+      content="summary_large_image"
+      name="twitter:card"
+    />
+    <meta
+      content="@BBCNews"
+      name="twitter:creator"
+    />
+    <meta
+      content="mostread"
+      name="twitter:description"
+    />
+    <meta
+      content="BBC News Pidgin"
+      name="twitter:image:alt"
+    />
+    <meta
+      content="https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"
+      name="twitter:image:src"
+    />
+    <meta
+      content="@BBCNews"
+      name="twitter:site"
+    />
+    <meta
+      content="De one we dem de read well well - BBC News Pidgin"
+      name="twitter:title"
+    />
+  </head>
+  <body>
+    <div>
+      <div>
+        chartbeat
+      </div>
```

Comment:
Should the banner/navigation be on here?

Reply:
I had a look at the other page snapshots and I couldn't see navigation/banner/footer in these.